.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,33 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
```

Comment:
I'm not sure I like this thing in here… this seems to be one of the things that every developer likes to have different.